consensus/reactor.go

```diff
@@ -169,6 +167,11 @@ conS:
 conR:
 %+v`, err, conR.conS, conR))
 	}
+	conR.propagator.StartProcessing()
+	conR.propagator.SetProposer(state.Validators.GetProposer().PubKey)
+	conR.conS.rsMtx.RLock()
+	conR.propagator.SetHeightAndRound(conR.conS.rs.Height, conR.conS.rs.Round)
```

Comment:
this sets the height and round in the propagation reactor to fix the case where consensus is start, then we replay some blocks, then we start from there. Also, this fixes the case where the chain starts at a higher chain.